src/Corax/Indexing/IndexWriter.IndexEntryBuilder.cs

```diff
@@ -421,7 +421,7 @@ void RegisterTerm(IndexedField field, ReadOnlySpan<byte> term, StoredFieldType t
                 out Span<byte> space);
             term.CopyTo(space);
 
-            var recordedTerm = RecordedTerm.CreateForStored(entryTerms, type, termId);
+            var recordedTerm = RecordedTerm.CreateForStored(entryTerms, type, (long)termId);
```

Comment:
Can you change `long`  into `ContainerId` in `RecordedTerm`? That would be more consistent.

Reply:
I would replace the current long parameter with `ContainerId`, just to make sure the usage is consistent across the codebase.